api/v1/registry.proto

```diff
@@ -58,6 +60,23 @@ message VersionedApplicationSpec {
     repeated Flavor flavors = 40;
 }
 
+// for use when an installation spec contains multiple steps
```

Comment:
1) We could define the behavior of this type of install spec to merge all of the manifests from all the steps into a single large manifest, apply the layers, then rely on the installer doing things in the right order. This may fail for istio because the init manifest creates jobs that create CRDs, so our installer won't know there are CRDs that need to come first. 
2) We could define the behavior such that any customizations with layers apply only to the last package. If all the packages require different layers, at that point they should be modeled as separate applications. In this case, the init package requires no customization, we just want to apply it first, then apply the real istio package with customizations.

Reply:
3. Go back to modeling as multiple application specs.

Reply:
We resolved this -- going to update the installer to grab all the manifests from all the steps, apply customizations for the flavor selected, then install step by step.